config/jinja2.py

```diff
@@ -257,6 +257,18 @@ def append_text_chunk(text, remaining_visible):
     return "".join(output)
 
 
+def override_tna_record_count(value, record):
+    """
+    Override the record count for specific 'held by' values.
+    """
+    if re.search(
+        r"The National Archives", (record.held_by or ""), re.IGNORECASE
+    ):
```

Comment:
Looks like there’s a hard-coded value here. It might be more flexible to use a constant or configuration instead. - What do you think about trying `record.is_tna` here? `is_tna` returns True/False if the record is a TNA or not.

Reply:
Thanks @TNA-Allan - that's an excellent suggestion, I'll push up an amended commit momentarily and let you know when it's ready for a review.